$NetBSD: patch-man_sshguard.8,v 1.1 2014/07/26 19:57:44 tron Exp $

Patches by wizd(8). Without these changes NetBSD's man(1) refuses to
render this manual page.

--- man/sshguard.8.orig	2011-02-09 12:01:47.000000000 +0000
+++ man/sshguard.8	2014-07-23 23:37:05.000000000 +0100
@@ -15,6 +15,7 @@
 .\"
 .Dd Mar 31, 2010
 .Dt SSHGUARD 8
+.Os
 .Sh NAME
 .Nm sshguard
 .Nd monitors daemon activity 
@@ -55,7 +56,7 @@
 .Pp
 .Nm
 supports the following firewalls:
-.Bl -tag -width
+.Bl -tag -width 20n
 .It AIX native firewall
 for IBM AIX operating systems
 .It netfilter/iptables
@@ -204,7 +205,7 @@
 .Fl w
 command-line option. This option can add explicit addresses, host names and
 address blocks:
-.Bl -tag -width
+.Bl -tag -width 30n
 .It addresses
 specify the numeric IPv4 or IPv6 address directly, like:
 .Dl -w 192.168.1.10
@@ -292,7 +293,7 @@
 policy (all of their log messages are accepted by default).
 .Pp
 PIDs are checked with the following policy:
-.Bl -enum -width
+.Bl -enum
 .It
 the logging service is searched in the list of services configured for
 validation. If not found, the entry is accepted.
